### PR TITLE
Day6

### PR DESCRIPTION
**Pull request by andreealung96**
Login, Evaluations, New Evaluation pages with JS

### CODE REVIEW

File: src/views/evaluations/evaluations.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        <td>${options.name}</td>
+        <td>${options.technology}</td>
+        <td>${options.level}</td>
+        <td style="text-align: right">Detalii <button>+</button></td>
+    </tr>
+    `
+}
+
+function EvaluationsTableBody (options = {}) {
+    //const rowsElements = [];
```
Comment (softvision-alexcarcea) [inline src/views/evaluations/evaluations.js:64]: Commented code goes away as soon as alternative is confirmed to work properly, yes? :stuck_out_tongue:

File: src/views/login/login.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    const body = document.querySelector("#app");
+    body.innerHTML = loginHTML();
+}
+
+
+
+
+
+
+/* const DropdownSelector = function (options){
```
Comment (softvision-alexcarcea) [inline src/views/login/login.js:89]: Can this be removed?

File: src/views/new-evaluation/new-evaluation.js [hunk 3]
```diff
@@ -0,0 +1,590 @@
...
+        ${CandidateDetailsForm()}
+        ${TechnicalLevelPicker()}
+        ${Textarea()}
+        ${Textarea()}
+        ${Textarea()}
+        <fieldset class="evaluation_options">
+            <legend class="evaluation_category">OOP, Design Patterns</legend>
+            <ul>
+                <li>Classes</li>
+                <li>
```
Comment (softvision-alexcarcea) [inline src/views/new-evaluation/new-evaluation.js:83]: What a big bunch of highly repetitive code here... If only there were ways of doing this so we didn't have to repeat ourselves all over the place... :stuck_out_tongue:

**Review by softvision-alexcarcea — APPROVED** [review]

File: src/views/evaluations/evaluations.js (hunk 1 above)
Comment (andreealung96) [reply src/views/evaluations/evaluations.js:64]: Right! :)

File: src/views/new-evaluation/new-evaluation.js (hunk 3 above)
Comment (andreealung96) [reply src/views/new-evaluation/new-evaluation.js:83]: Yep, I still have to take care of this and generate the elements using JS. Haven't had the time to finish it yesterday.

**Pull request merged**